about/index.html

```diff
@@ -19,17 +19,38 @@
           >
         </a>
       </div>
+      <menu class="index">
+        <span> Uncanny Index ↓</span>
+        <ul class="dropdown-content">
+          <li><a href="../self-fulfilling-prophesy/index.html">Self Fulfilling Prophecy</a></li>
+          <li><a href="../impostor-syndrome/index.html">Impostor Syndrome</a></li>
+          <li><a href="../fermi-paradox/index.html">Fermi Paradox</a></li>
+          <li><a href="../phenomenon-of-baader-meinhof/index.html">Phenomenon of Baader-Meinhof</a></li>
+          <li><a href="../curse-of-knowledge/index.html">Curse of Knowledge</a></li>
+          <li><a href="../stockholm-syndrome/index.html">Stockholm Syndrome</a></li>
+          <li><a href="../uncanny-valley/index.html">Uncanny Valley</a></li> 
+          <li><a href="../observer-expectancy-effect/index.html">Observer-expectancy effect</a></li>
+          <li><a href="../base-rate-fallacy/index.html">Base Rate Fallacy</a></li>
+        </ul>
+      </menu>
     </header>
 
-    <div class="line"></div>
-
-    <main>
-      <h1>About me</h1>
-      <p></p>
+    <main id="about">
+       <div class="about-content">
+        <ul>
+          <li><img src="../assets/susu.jpg"> Non-binary trans person (he/they pronouns)</li>
+          <li><img src="../assets/susu.jpg"> Argentinian</li>
+          <li><img src="../assets/susu.jpg"> Socially awkward</li>
+          <li><img src="../assets/susu.jpg"> Takes care of plants</li>
+          <li><img src="../assets/susu.jpg"> Always wants to play boardgames but hardly ever does so</li>
+          <li><img src="../assets/susu.jpg"> Has a lot of books that didn't read</li>
+          <li><img src="../assets/susu.jpg"> Likes lists (obviously)</li>
+          <li><img src="../assets/susu.jpg"> Hates passive-agressive comunication</li>
+          <li><img src="../assets/susu.jpg"> Perfect date is sharing silence with a friend</li>
```

Comment:
Don't leave empty spaces inside elements between the tag and the content, they get removed by the browser (unless you want a space there for design reasons, in that case you need to do it differently).